estate/models/estate_property_type.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class estate_property_type(models.Model):
+    _name='estate.property.type'
```

Comment:
```suggestion
    _name = 'estate.property.type'
```
exactly one space required before and after while doing operation on variable